atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/command/TestTimestampCommand.java

```diff
@@ -163,8 +163,6 @@ public void testWithoutFile() throws Exception {
             String output = runner.run(true, false);
             try {
                 Scanner scanner = new Scanner(output);
-                String shouldBeDeprecationLine = scanner.nextLine();
-                verifyDeprecation(shouldBeDeprecationLine);
```

Comment:
This deprecation line is [printed at the warn level](https://github.com/palantir/atlasdb/blob/0e5f3fbe8d90e3d00c4d2330a1606570f40173ec/atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/FetchTimestamp.java#L63). The [`OutputPrinter`](https://github.com/palantir/atlasdb/blob/develop/atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/output/OutputPrinter.java) that writes these logs lines writes them to both stdout/stderr and the configured logger.

The test passes today because of the logback dependency, which by default emits warn logs to stdout. This causes warning and error logs to be emitted twice - to both stdout and stderr. After removing that dependency warn logs are correctly emitted to only stderr.

We could wire this up to capture stdout and stderr. But I don't think this assertion provides much value so I've just removed it.